docs/platform/visual/index.md

```diff
@@ -34,21 +33,17 @@ Visualizations are implemented on top of the Pentaho Core, Type and Data JavaScr
   as well as with other client-side component frameworks.
 - Using [Core APIs]()
   provides visualizations with features such as localization, theming and registration.
-  
-The use of [Pentaho Platform JavaScript APIs]({{site.refDocsUrlPattern | replace: '$', 'pentaho'}}) for anything not 
-directly related with the visualization concept, 
-ensures developers can reuse knowledge that they gain in other components of the Pentaho suite 
-to build visualizations and vice-versa.
 
-The Pentaho Visualization API comes with a set of stock visualizations, 
-covering the most common chart-types.
-Most stock visualizations are based on the [CCC](http://community.pentaho.com/ctools/ccc/) charting library,
-which means that the stock visualizations are _super-configurable_ to suite your organization's preferred style.
+The platform provides a set of stock visualizations, covering the most
+common chart-types.
+Based on the [CCC](http://community.pentaho.com/ctools/ccc/) charting
+library, they're customizable and extensible to fit your organization's
+desired look and feel.
```

Comment:
**Super** configurable seems a bit self praising. Maybe a bit less marketing? And should we link to configuration documentation, specially if we talk about where to put the CCC extension points?